Server/datalayer/time-entry.datalayer.js

```diff
@@ -1,12 +1,111 @@
+const timeEntryModel = require('../models/time-entry.model');
+const moment = require('moment');
+const mongoose = require('mongoose');
 
 /*
  * @class timeEntriesDatalayer
  */
 class timeEntriesDatalayer {
 
     /**
-    * Initializes the datalayer
-    */
+     * Initializes the datalayer
+     */
+    static getTimeEntriesByUserId(obj) {
+
+        //if user id not given in URL, send error message
+        if (obj.userId == undefined){
+            throw new Error('No User ID provided');
+        }   
+
+        //set default sort order
+        let sortOrder = (obj.sortOrder == 'asc') ? { '_id': 1 } : { '_id': -1 };
```

Comment:
Change it to a switch case as more keywords can come
`switch(obj.sortOrder{
case 'asc':
case 'desc':
default:
})`

Reply:
Updated